renderdoc/driver/gl/gl_enum.h

```diff
@@ -2428,6 +2428,7 @@ enum RDCGLenum
   eGL_ABGR_EXT = 0x8000,
   eGL_BGR_EXT = 0x80E0,
   eGL_BGRA_EXT = 0x80E1,
+  eGL_BGRA8_EXT = 0x93A1,
```

Comment:
This file is autogenerated from the official headers, so manually adding enums here will get overwritten in future.

Instead it's better to re-run the generation script at the top of the file to pick up any new enums - I guess the script wasn't re-run to pick up the new enums since the GLES headers were committed.